deployment/terraform/ses.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_ses_domain_identity" "app" {
+  domain = "${var.r53_public_hosted_zone}"
+}
+
+resource "aws_ses_domain_dkim" "app" {
```

Comment:
This is interesting. Until now, we haven't codified this process. Interested to see how it plays out.

Reply:
Me too. Seems to have worked well so far.